Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hydroshare/hs_docker_base:cb5fb73
+FROM hydroshare/hs_docker_base:41f2084
```

Comment:
I assume this is for the celery/setup_tools updates

Reply:
that's correct